src/config/script/list_values.py

```diff
@@ -52,6 +52,46 @@ def __init__(self, script) -> None:
     def get_values(self, parameter_values):
         return self._values
 
+class DependantValuesProvider(ValuesProvider):
```

Comment:
Would you mind adding a couple of tests for this class? Specifically for `get_values` method